lib/pure/parsesql.nim

```diff
@@ -689,9 +702,15 @@ proc parseSelect(p: var SqlParser): SqlNode
 
 proc identOrLiteral(p: var SqlParser): SqlNode =
   case p.tok.kind
-  of tkIdentifier, tkQuotedIdentifier:
+  of tkQuotedIdentifier:
     result = newNode(nkIdent, p.tok.literal)
     getTok(p)
+  of tkIdentifier:
+    if p.tok.literal.toLowerAscii() in SqlFunctions:
```

Comment:
Er, that is not how the SQL is supposed to work. We don't know the SQL functions there can be hundreds per SQL dialect. Why special case these few? It also makes the parser slower, ``tkIdentifier`` is likely on the critical path.

Reply:
I removed the functions list, but I had to pass Quoted Identifiers from tokens to nodes as there needs to be a way to say quote `foo as "count"` and also have the unquoted `count(*)`.